README.md

```diff
@@ -59,7 +59,7 @@ different resources yourself.
 
 * Check the [Recipes](#language-recipes) list below on how to configure these:
 * Node.js (any version via [nave](https://github.com/isaacs/nave))
-* Python ([3.8.0, 3.7.4, 3.6.9](#python))
+* Python ([3.9.7, 3.8.12, 3.7.12, 3.6.15](#python))
```

Comment:
These are the latest version of each Python sub-version at the point of creating this PR.